nextjs-logs-manager/README.md

```
@@ -1,36 +1,44 @@
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+### Populate the database with test users
+Open your browser and navigate to the following URL, or simply <a href="http://localhost:3000/api/seed" target="_blank">Click here</a> to populate the database with test users.
+```
+http://localhost:3000/api/seed
+```
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+## Users Credentials
+### Admin
+- email: admin@test.com
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)